src/components/cards/TradeCard/GasReimbursement.vue

```diff
@@ -7,9 +7,7 @@
     <div v-if="isActive()" class="message relative px-2 py-3">
       <div class="ml-12">
         <h6 v-text="$t('highGasFees')" class="relative text-sm" />
-        <div class="relative text-sm text-gray-500">
-          {{ text }}
-        </div>
+        <div class="relative text-sm text-gray-500" v-html="text" />
```

Comment:
That is absolutely cool to have HTML working like that with the simple `v-html` tag!

Reply:
If it's just text and not html in the variable `text` you can just use `v-text` instead.